unstructured/ingest/cli/interfaces.py

```diff
@@ -468,13 +468,13 @@ def get_cli_options() -> t.List[click.Option]:
                 default=False,
             ),
             click.Option(
-                ["--chunk-combine-under-n-chars"],
+                ["--chunk-combine-text-under-n-chars"],
                 type=int,
                 default=500,
                 show_default=True,
             ),
             click.Option(
-                ["--chunk-new-after-n-chars"],
+                ["--chunk-max-characters"],
```

Comment:
i think we need to support `max_characters` in addition to `new_after_n_chars`. i.e. we should keep new-after and add max-characters 

note, these are not the same thing:


    new_after_n_chars
        Cuts off new sections once they reach a length of n characters (soft max). Defaults to
        `max_characters` when not specified, which effectively disables any soft window.
        Specifying 0 for this argument causes each element to appear in a chunk by itself (although
        an element with text longer than `max_characters` will be still be split into two or more
        chunks).
    max_characters
        Chunks elements text and text_as_html (if present) into chunks of length
        n characters (hard max)

Reply:
Agreed, addressed with commit: [add support for new_after_n_chars](https://github.com/Unstructured-IO/unstructured/pull/1774/commits/1e8f34e9c5e48fec933ed47290b571703afd4552)

Ingest test for the pinecone connector is also updated to test if new_after_n_chars works as intended